packages/create-preact-app/tests/create.test.js

```diff
@@ -37,10 +37,4 @@ describe('preact create', () => {
 
 		expect(template.includes('apple-touch-icon')).toEqual(true);
 	});
-
-	// it('should fail given an invalid name', async () => {
-	// 	const exit = jest.spyOn(process, 'exit');
-	// 	await create('default', '*()@!#!$-invalid-name');
-	// 	expect(exit).toHaveBeenCalledWith(1);
-	// });
```

Comment:
Removed as this has been commented out and unused for 3 years: https://github.com/preactjs/preact-cli/commit/fef996e7ee55c58969ba52557d33e8e5bde81f99